app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/edit.hbs

```diff
@@ -0,0 +1,278 @@
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuHeading>
+        Bewerk mandataris
+      </AuHeading>
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-top-tiny">
+        {{@model.person.givenName}}
+        {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Card|>
+        <Card.Columns>
+          <:left as |Item|>
+            <Item>
+              <:label>Naam</:label>
+              <:content>
+                {{@model.person.givenName}}
+                {{@model.person.familyName}}
+              </:content>
+            </Item>
+          </:left>
+
+        </Card.Columns>
+      </:card>
+    </DataCard>
+    <form
+      id="edit-mandatory-form"
+      aria-label="edit-mandatory-form"
+      {{on "submit" (perform this.save)}}
+    >
+      <div class="au-o-flow au-o-flow--large">
+        <EditCard>
+          <:title>Positie</:title>
+          <:card as |Card|>
+            <Card.Columns>
+              <:left as |Item|>
+                <Item>
+                  <:label>Type</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value="Mandaat"
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+                <Item>
+                  <:label>Bestuursfunctie</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{this.mandatory.mandate.roleBoard.label}}
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+              </:left>
+              <:right as |Item|>
+                <Item>
+                  <:label>Naam Eredienst</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{this.administrativeUnit.name}}
+                      disabled={{true}}
+                    />
+                  </:content>
+                </Item>
+              </:right>
+            </Card.Columns>
+          </:card>
+        </EditCard>
+
+        <EditCard>
+          <:title>Primaire contactgegevens</:title>
+          <:card as |Card|>
+            <Card.Columns>
+              <:left as |Item|>
+                <Item @labelFor="contact-email">
+                  <:label>Email</:label>
+                  <:content>
+                    <AuInput
+                      @width="block"
+                      @value={{this.mandatory.contacts.firstObject.email}}
+                      id="contact-email"
+                    />
+                  </:content>
+                </Item>
+                {{#if this.mandatory.contacts.firstObject.contactAddress.fullAddress}}
+                  <Item>
```

Comment:
I don't think this one is needed? The user needs to edit the individual fields right?

Reply:
This will be useful in the beginning because the address is only there, so it acts as a helper to see the address and then split it in each field.

Reply:
ah, right, didn't notice this wasn't an input field.